components/script/dom/svgelement.rs

```diff
@@ -129,4 +130,15 @@ impl SVGElementMethods<crate::DomTypeHolder> for SVGElement {
         self.element
             .set_bool_attribute(&local_name!("autofocus"), autofocus, can_gc);
     }
+
+    // https://html.spec.whatwg.org/multipage/#dom-focus
+    fn Focus(&self, options: &FocusOptions) {
+        let document = self.element.owner_document();
+        document.request_focus_with_options(
+            Some(&self.element),
+            FocusInitiator::Local,
+            options.preventScroll,
+            CanGc::note(),
+        );
+    }
```

Comment:
It would be nice to elaborate why are we not following the steps by steps in https://html.spec.whatwg.org/multipage/interaction.html#dom-focus for `SVGElement::Focus` and `HTMLElement::Focus` .